cpp/modmesh/view/RManager.cpp

```diff
@@ -153,17 +153,43 @@ void RManager::setUpMenu()
         auto * use_orbit_camera = new RAction(
             QString("Use Orbit Camera Controller"),
             QString("Use Oribt Camera Controller"),
-            []()
+            [this]()
             {
                 qDebug() << "Use Orbit Camera Controller (menu demo)";
+                for (auto subwin : m_mdiArea->subWindowList())
```

Comment:
This duplicates the code at line 180.  In the future we should factor it out to be a common helper.